File: baekjoon/bronze/level2/test_1978/java/Main.java
package baekjoon.bronze.level2.test_1978.java;

import java.io.*;
import java.util.stream.Stream;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

        int n = Integer.parseInt(br.readLine());
        int[] arr = Stream.of(br.readLine().split(" ")).mapToInt(Integer::parseInt).toArray();
        int count = 0;

        for (int i = 0; i < n; i++) {
            count += arr[i] > 1 ? isPrime(arr[i]) : 0;
        }

        bw.write(String.valueOf(count));
        bw.close();
    }

    public static int isPrime(int n) {
        for (int i = 2; i<=(int)Math.sqrt(n); i++) {
            if (n % i == 0) {
                return 0;
            }
        }
        return 1;
    }
}

File: baekjoon/bronze/level2/test_1978/kotlin/Main.kt
package baekjoon.bronze.level2.test_1978.kotlin

fun main() {
    val n = readln().toInt()
    val arr = readln().split(" ").map { it.toInt() }

    var answer = 0;
    for (i in 0..<n) {
        val num = arr[i]
        var count = 0;

        for (j in 2..num) {
            if (num % j == 0) {
                count++
            }
        }

        if (count == 1) {
            answer++
        }
    }
    print(answer)
}
